snap/snapcraft.yaml

```diff
@@ -0,0 +1,63 @@
+name: tempest
+adopt-info: tempest
+summary: OpenStack Integration Test Suite
+description: |
+ Tempest is a set of integration tests to be run against a live
+ OpenStack cluster. Tempest has batteries of tests for OpenStack
+ API validation, Scenarios, and other specific tests useful in
+ validating an OpenStack deployment.
+confinement: strict
+grade: stable
+base: core22
+
+apps:
+  tempest:
+    command: bin/tempest
+    completer: usr/share/bash-completion/completions/tempest
+    plugs:
+      - network-observe
+      - network
+      - home
+parts:
+  tempest:
+    plugin: python
+    source: .
+    python-packages:
+      - tempest
```

Comment:
I think we should use versions here or not? For tempest and all plugins, so we can do l release for each supported tempest and openstack version.
Maybe here we can use local requirements.txt or provide external one. I'm thinking at loud only how we can do auto release.